src/components/Gallery/GridImage.tsx

```diff
@@ -28,6 +28,14 @@ export const GridImage: FunctionComponent<GridImageComponentProps> = ({
   const handleOnImageError = () => {
     setImageDidError(true);
   };
+
+  const VideoThumbnail = () => {
+    const url =
+      asset.src.replace('imgix.net', 'imgix.video') +
+      '?video-generate=thumbnail&time=0.1';
```

Comment:
Do we need to add a `time` here? I think it defaults to 1/2 way through the video if memory serves.

Reply:
We don't, this is just to mimic the current behavior in dashboard.